backend/src/libs/packages/config/libs/interfaces/config.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { type Config as LibraryConfig } from 'shared/build/index.js';
+
+import { type EnvironmentSchema } from '../types/types.js';
+
+interface Config extends LibraryConfig<EnvironmentSchema> {}
```

Comment:
Maybe type instead?

Reply:
we use it to implements class from it, that's why

Reply:
UPD: Replaced with types